src/main/java/me/juancarloscp52/entropy/client/Screens/Widgets/EntropyEventListWidget.java

```diff
@@ -119,27 +120,6 @@ protected ButtonEntry getEntryAtPositionRespectingSearch(double x, double y) {
         return x < (double)this.getScrollbarPositionX() && x >= (double)k && x <= (double)l && n >= 0 && m >= 0 && n < this.getEntryCount() ? this.visibleEntries.get(n) : null;
     }
 
-    @Override
-    protected void renderList(DrawContext drawContext, int mouseX, int mouseY, float delta) {
-        int rowLeft = this.getRowLeft();
-        int rowWidth = this.getRowWidth();
-        int entryHeight = this.itemHeight - 4;
-        int entryCount = this.children().size();
-        int drawIndex = 0;
-
-        for (int index = 0; index < entryCount; ++index) {
-            int rowTop = this.getRowTop(drawIndex);
-            int rowBottom = rowTop + this.itemHeight;
-
-            if (this.getEntry(index).checkbox.visible) {
-                drawIndex++;
-
-                if (rowBottom >= this.top && rowTop <= this.bottom)
-                    this.renderEntry(drawContext, mouseX, mouseY, delta, index, rowLeft, rowTop, rowWidth, entryHeight);
-            }
-        }
-    }
-
```

Comment:
Just realized the removal of this code is likely the cause of my second point.

Reply:
Yes, also just found that.